webapp/src/onedrive.ts

```diff
@@ -417,6 +417,7 @@ export class Provider extends cloudsync.ProviderBase implements cloudsync.Provid
                 ifr.style.border = "0";
                 hiddenFrame = (document.getElementsByTagName("body")[0].appendChild(ifr) as HTMLIFrameElement);
             } else if (document.body && document.body.insertAdjacentHTML) {
+                // eslint-disable-next-line @microsoft/sdl/no-inner-html
```

Comment:
This seems to be a new disable. Is this safe in this context?

Reply:
Inner html are risky

Reply:
I'll review it. We are using innerHtml in ~12 places :/

Reply:
@pelikhan who have found issues here in the past.